sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-data-inventory-report.sql

```diff
@@ -0,0 +1,505 @@
+INSERT INTO
+    sims.report_configs (report_name, report_sql)
+VALUES
+    (
+        'Data_Inventory_Report',
+        'select
+            -- Personal Info
+            users.first_name as "First Name",
+            users.last_name as "Last Name",
+            students.birth_date as "Date of Birth",
+            students.gender as "Gender",
+            sin_validations.sin as "SIN",
+            users.email as "Email address",
+            students.contact_info -> ''address'' ->> ''addressLine1'' as "Address Line 1",
+            students.contact_info -> ''address'' ->> ''addressLine2'' as "Address Line 2",
```

Comment:
Minor. Do we need a default filler for nullable values? Is it something we are not considering in this ticket?

Reply:
I thought about but I didn't consider that in this ticket.

Reply:
I added a question on #2580